lib/node_modules/@stdlib/lapack/base/dlaev2/README.md

```diff
@@ -0,0 +1,235 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dlaev2
+
+> Compute the eigendecomposition of a 2x2 symmetric matrix.
+
+<section class = "usage">
+
+## Usage
+
+```javascript
+var dlaev2 = require( '@stdlib/lapack/base/dlaev2' );
+```
+
+#### dlaev2( A, B, C, out, ev )
+
+Computes the eigendecomposition of a 2x2 symmetric matrix.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var ev = new Float64Array( 2 );
+var out = new Float64Array( 2 );
+out = dlaev2( 2.0, 3.0, 4.0, out, ev );
+// out => <Float64Array>[ ~6.162, ~-0.162 ]
+// ev => <Float64Array>[ ~0.585, ~0.811 ]
+```
+
+The function has the following parameters:
+
+-   **A**: the (0,0) element of a 2x2 symmetric matrix.
+-   **B**: the (0,1) and the conjugate of (1,0) element of a 2x2 symmetric matrix.
+-   **C**: the (1,1) element of a 2x2 symmetric matrix.
+-   **out**: output [`Float64Array`][mdn-float64array] containing the eigenvalues of larger and smaller absolute values respectively.
+-   **ev**: output [`Float64Array`][mdn-float64array] containing `CS1` and `SN1` which is unit right eigenvector for `RT1` giving the decomposition.
```

Comment:
You're going to need to explain `CS1` and `SN1` and `RT1`. This description doesn't tell me a whole lot. Either add process or show equations.